File: ProgLab9.py
def init():
    global line
    global wordLength
    line = input("Enter line: ")
    wordLength = int(input("Enter word length: "))
def getWords(line):
    global words
    words = line.split()
    return words
def displayWords(words, length):
    i = 0
    while(i < len(words)):
        if(len(words[i]) == length):
            print(words[i])
        i += 1
init()
displayWords(getWords(line), wordLength)

File: ProgLab9.cs
using System;

namespace ProgLab9
{
    class Program
    {
        static string line;
        static int wordLength;

        static void Main(string[] args)
        {
            Init();
            DisplayWords(GetWords(line), wordLength);
        }

        static void Init()
        {
            Console.WriteLine("Enter line: ");
            line = Console.ReadLine();
            Console.WriteLine("Enter word length: ");
            wordLength = Convert.ToInt32(Console.ReadLine());
        }

        static string[] GetWords(string line)
        {
            string[] words = line.Split(" ");
            return words;
        }

        static void DisplayWords(string[] words, int length)
        {
            for(int i = 0; i < words.Length; i++)
            {
                if (words[i].Length == length)
                    Console.WriteLine(words[i]);
            }
        }

    }
}
